src/main/java/miro/task/ToDoTask.java

```diff
@@ -0,0 +1,18 @@
+package miro.task;
+
+public class ToDoTask extends Task {

```

Comment:
```suggestion
public class TodoTask extends Task {
```
Per naming rules, abbreviations and acronyms should not be uppercase when used as a part of a name.